project/Dependencies.scala

```diff
@@ -19,6 +19,7 @@ object Dependencies {
   lazy val spark_sql = "org.apache.spark" %% "spark-sql" % "2.4.5" % "provided"
   lazy val postgres_conn = "org.postgresql" % "postgresql" % "9.3-1100-jdbc4"
   lazy val junit = "org.junit.jupiter" % "junit-jupiter-api" % "5.3.2" % Test
+  lazy val mongo = "org.mongodb.scala" %% "mongo-scala-driver" % "2.8.0" % "provided"
```

Comment:
Let's remember to state that the MongoDB driver is not provided in the dependencies in the README.md